docs/source/actions/merge.rst

```diff
@@ -171,6 +172,19 @@ request:
 Defining the Commit Message
 ---------------------------
 
+|deprecated tag|
+
+.. warning::
+
+   `commit_message` will be deprecated in April 25th, 2022. The :ref:`data type template`
+   will provide a more powerful solution
+   and offer total control of the merge commit.
```

Comment:
Well if you mark it deprecated today, it's not going to be deprecated on 25th April.
I guess you mean "removed" here, not deprecated.